clash-starters/clash-example-project/stack.yaml

```diff
@@ -0,0 +1,12 @@
+resolver: lts-15.10
+
+extra-deps:
+- concurrent-supply-0.1.8
+- ghc-typelits-extra-0.4
+- clash-prelude-1.2.2
+- clash-lib-1.2.2
+- clash-ghc-1.2.2
+
+flags:
+  clash-prelude:
+    large-tuples: false
```

Comment:
We want to disable multiple implicit clocks as well, right?

Reply:
That's already disabled by default for releases :)

Reply:
I'll add a comment why we disable it though

Reply:
ah, alright